packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -743,6 +750,11 @@ export class Function extends FunctionBase {
     }
     this._architecture = props.architecture ?? (props.architectures && props.architectures[0]);
 
+    if (props.ephemeralStorageSize && !Token.isUnresolved(props.ephemeralStorageSize)
```

Comment:
Was just going to comment this :). `Size.isUnresolved()` is introduced in [this](https://github.com/aws/aws-cdk/pull/19569) PR that is almost in. 

Btw I asked for testing for token support in this PR because it is a possible use case and a common bug we introduce in the CDK. Any time we are validating something we should think about the use case when a token is involved.

Reply:
I merged in this change but now I'm getting this error:

>  The declaring class is introduced here
lib/function.ts(757,67): error TS2339: Property 'isUnresolved' does not exist on type 'Size'.

Reply:
I just saw PR @kaizen3031593  mentioned... I'll wait until that gets merged in.